File: src/targets/swift/urlsession/fixtures/cookies.swift
import Foundation
#if canImport(FoundationNetworking)
  import FoundationNetworking
#endif

let headers = ["cookie": "foo=bar; bar=baz"]

var request = URLRequest(url: URL(string: "https://httpbin.org/cookies")!)
request.httpMethod = "GET"
request.allHTTPHeaderFields = headers

let (data, response) = try await URLSession.shared.data(for: request)
print(String(decoding: data, as: UTF8.self))

File: src/targets/rust/reqwest/fixtures/cookies.rs
use reqwest;

#[tokio::main]
pub async fn main() {
    let url = "https://httpbin.org/cookies";

    let client = reqwest::Client::new();
    let response = client.get(url)
        .header("cookie", "foo=bar; bar=baz")
        .send()
        .await;

    let results = response.unwrap()
        .json::<serde_json::Value>()
        .await
        .unwrap();

    println!("{}", results);
}
